Source/Dafny/ConcreteSyntax/ConcreteSyntaxTree.cs

```diff
@@ -114,6 +115,12 @@ public T Append<T>(T node)
       return this;
     }
 
+    public void DeleteLast() {
```

Comment:
Very glad we don't need this any more. Now let's remove the method so no one is tempted by the dark side in the future. :)